server/mdm/nanomdm/storage/mysql/async.go

```diff
@@ -32,13 +32,17 @@ func (a *asyncLastSeen) markHostSeen(ctx context.Context, id string) {
 }
 
 func (a *asyncLastSeen) runFlushLoop(ctx context.Context) {
-	tickCh := time.Tick(a.flushInterval)
+	ticker := time.NewTicker(a.flushInterval)
```

Comment:
`time.NewTicker` provides granular control over the ticker's lifecycle, whereas `time.Tick` is a [convenience wrapper](https://pkg.go.dev/time#Tick) that does not.